### PR TITLE
Microsoft365 DSC for Tenant Admin - Intune Derived Credentials.

### PR DESCRIPTION
**Pull request by Kajalp1079**

### CODE REVIEW

File: Modules/Microsoft365DSC/DSCResources/MSFT_IntuneDerivedCredential/settings.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+    "resourceName": "IntuneDerivedCredential",
+    "description": "Use this resource to create new navigation property to derivedCredentials for device Management in Intune.",
+
+    "permissions": {
+      "graph": {
+        "delegated": {
+          "read": [
+            ""
```
Comment (NikCharlebois) [inline Modules/Microsoft365DSC/DSCResources/MSFT_IntuneDerivedCredential/settings.json]: Either leave this empty [] or provide a value but empty quotes is making the tests fail.

**Review by NikCharlebois — CHANGES_REQUESTED** [review]
Small change requested in the settings.json permissions

File: Modules/Microsoft365DSC/DSCResources/MSFT_IntuneDerivedCredential/settings.json (hunk 1 above)
Comment (Kajalp1079) [reply Modules/Microsoft365DSC/DSCResources/MSFT_IntuneDerivedCredential/settings.json]: updated and pushed.

**Comment by NikCharlebois** [comment]
Also, please make sure you merge with upstream to get the latest fixes to the unit tests:

git merge upstream

**Review by NikCharlebois — APPROVED** [review]

**Pull request merged**